packages/ui/src/stories/Typography.stories.mdx

```diff
@@ -0,0 +1,139 @@
+import { Meta } from '@storybook/addon-docs';
+
+<Meta title="Design Tokens/Typography" />
+
+# Typography
+
+This Design System leverages a purposeful set of typographic styles. This typography scale is robust enough to use across (almost) any newsroom, while remaining as accessible as possible for everyone.
+
+<a
+    className="sb-unstyled text-indigo-600"
+    href="https://www.figma.com/file/46dEAasj1iEtrVQOwmMswB/00--%3E-Themes-Design-System?type=design&node-id=612-13936&mode=dev"
+    target="_blank"
+    rel="noopener noreferrer"
+>
+    See in Figma Design System
+</a>
+
+<br />
+<br />
+
+<div className="mt-8">
+    <p className="title-x-large sb-unstyled">Title xLarge</p>
```

Comment:
It seems you didn't see my updated comment, I initially thought that you'd have to attach the class to every text node, but you can also wrap the whole part with examples in a div with `sb-unstyled` class. That will make adding/editing the examples easier.

Reply:
Cool, will make the update